deploy/ansible/playbook.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+# Run this playbook via e.g.: ansible-playbook -i inventory-staging.yaml playbook.yml
+- name: Build and Deploy Poseidon
+  connection: ssh
+  gather_facts: false
+  hosts: all
+  tasks:
+    # Build Poseidon
+    - name: Build Poseidon
+      local_action:
+        module: make
+        chdir: "{{ playbook_dir }}/../.."
+        target: build
```

Comment:
Currently we compile Poseidon on the local computer / controller. Although this has the advantage that we do not need all the files and dependencies required for compiling on the Poseidon server, this can also break down under certain architecture combinations...

Reply:
I am fine with compiling Poseidon locally and pushing the binary to the server as part of the deployment process.